src/imitation/algorithms/adversarial.py

```diff
@@ -225,6 +226,26 @@ def train_disc_step(self, *,
         logger.logkv(k, v)
       logger.dumpkvs()
 
+  def eval_disc_loss(self, **kwargs) -> float:
+    """Evaluates the discriminator loss.
+
+    The generator rollout parameters of the form "gen_*" are optional,
```

Comment:
there's only one keyword argument beginning with `gen_*` documented below? confused

Reply:
Hmm I thought I had deleted this part of the documentation a while ago. Maybe I made a bad merge sometime back.